recipe/meta.yaml

```diff
@@ -24,24 +26,30 @@ requirements:
   host:
     - grpcio =={{ version }}
     - pip
-    - protobuf >=3.6.0
+    - setuptools
+    - wheel
+    - protobuf
```

Comment:
libprotobuf is pinned in the cbc, but not protobuf. In that case I would still add the corresponding pinning.

Reply:
Ah! Good catch. Thanks!

Reply:
I think it should be 3.20.3 (as that's what is pinned on cbc and being built for 3.11)

Reply:
Thanks. I'm having a day 😵